myapp/.gitignore

```diff
@@ -6,4 +6,5 @@
 yarn-debug.log*
 .yarn-integrity
 tmp/
-log/
\ No newline at end of file
+log/
+.byebug_history
```

Comment:
[nits] Recommend turn this on
<img width="500" alt="image" src="https://github.com/Fablic/training/assets/101234015/d936a309-f200-49bb-a0ed-293bf1daf3c4">